### DISCUSSION TITLE
Area quantification of a region (atlas)

### DISCUSSION THREAD

**Discussion by shuertaspenen** [simnibs/simnibs#258] [9. Results and analysis] [answered]
Hi, 
I'm trying to quantify the area of a specific region when we use an atlas. I know the number of triangles in my region, but I don't know how to obtain the sizes of each triangle or to obtain the area. Is it possible to do so? If yes, how? 

Thank you for your help,
Silvana

**Comment by jduemose** [comment, marked as answer]
Hi Silvana

If you are using python to process your results then load the mesh file and use its method `elements_volumes_and_areas`. It is used in [this](https://simnibs.github.io/simnibs/build/html/tutorial/analysis.html) tutorial for example. Similarly, you can also get node-wise area/volume using `nodes_volumes_or_areas`.

Best,
Jesper
